integration-tests/api/__tests__/admin/product.js

```diff
@@ -381,6 +381,30 @@ medusaIntegrationTestRunner({
           )
         })
 
+        it("returns a list of products where id is a list", async () => {
+          const response = await api
+            .get(
+              `/admin/products?id[]=${baseProduct.id},${proposedProduct.id}`,
```

Comment:
curious: Does this work, i thought it was supposed to be like this: 

```suggestion
              `/admin/products?id[]=${baseProduct.id}&id[]=${proposedProduct.id}`,
```

Reply:
@riqwan yup it does 👍